fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraSearchIT.java

```diff
@@ -216,7 +216,7 @@ public void testModifiedGreaterAndLessThan() throws Exception {
         assertEquals(1, resources.size());
         final var externalFedoraId = resources.get(0);
         final var lessThanNow = MODIFIED + encode("<") + earlier;
-        final var greaterThanNow = MODIFIED + encode(">") + now;
+        final var greaterThanNow = MODIFIED + encode(">") + earlier;
```

Comment:
I'm curious what this change was for? Also, is `now` not used anymore? It may be related to the failure `FedoraSearchIT.testModifiedGreaterAndLessThan:230 expected no results ==> expected: <0> but was: <1>`